src/lib/ledger-context.js

```diff
@@ -35,11 +35,17 @@ function parseAndValidateLedgerUrls (metadataUrls) {
 class LedgerContext {
   constructor (host, ledgerMetadata) {
     this.host = host
+    this.ledgerMetadata = ledgerMetadata
     this.urls = parseAndValidateLedgerUrls(ledgerMetadata.urls)
     debug('using service urls:', this.urls)
     this.prefix = ledgerMetadata.ilp_prefix
-    this.info = {
```

Comment:
The prefix may be overwritten, so the info would change: https://github.com/interledgerjs/ilp-plugin-bells/blob/master/src/lib/plugin.js#L212-L214